tests/20-functional.test

```diff
@@ -1,469 +0,0 @@
-#!tests/.venv/bin/python
```

Comment:
Awesome. I'm glad to not have these here.

> On Apr 14 2016, at 8:35 am, Francesco Banconi
&lt;notifications@github.com&gt; wrote:  

>

> In [tests/20-functional.test](https://github.com/juju/juju-gui-
charm/pull/59#discussion_r59708677):

>  
>  
>     > @@ -1,469 +0,0 @@

>     > -#!tests/.venv/bin/python

>  
>

> Yes we are.

>

> —  
You are receiving this because you commented.  
Reply to this email directly or [view it on GitHub](https://github.com/juju
/juju-gui-charm/pull/59/files/92cb8498385672757523062780ae1a85f097128e#r597086
77)![](https://github.com/notifications/beacon/AABoaF5_UWk1_3LG-
1zP8EoV_7uO8o_Pks5p3jR9gaJpZM4IHS48.gif)